invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/DeleteCommunityModal.js

```diff
@@ -43,8 +56,29 @@ export class DeleteCommunityModal extends Component {
 
   componentWillUnmount() {
     this.cancellableDelete && this.cancellableDelete.cancel();
+    this.cancellableMembersCountFetch && this.cancellableMembersCountFetch.cancel();
+    this.cancellableRecordsCountFetch && this.cancellableRecordsCountFetch.cancel();
   }
 
+  updateCommunityMetrics = async () => {
+    this.setState({ loading: true });
+    const { community } = this.props;
+    this.cancellableMembersCountFetch = withCancel(http.get(community.links.members));
+    this.cancellableRecordsCountFetch = withCancel(http.get(community.links.records));
+
+    try {
+      const membersResponse = await this.cancellableMembersCountFetch.promise;
+      const recordsResponse = await this.cancellableRecordsCountFetch.promise;
+      this.setState({
+        loading: false,
+        membersCount: membersResponse.data.hits.total,
+        recordsCount: recordsResponse.data.hits.total,
+      });
+    } catch (error) {
+      console.error(error);
```

Comment:
I don't think it is enough... it will not display anything to the user, no?
We should set the `error` with the message in the state (and test to see how it looks).